[Bugfix] useAzureCommunicationCallAdapter multiple call client fix (#3157)
Azure/communication-ui-library: packages/storybook/stories/Troubleshooting.stories.mdx

This issue will occur when using `@azure/communication-react` with a webpack version < 5.
The best course of action is to upgrade your webpack version to 5.
If this is not possible, take a look at [these workarounds](https://github.com/fb55/htmlparser2/issues/1237#issuecomment-1182522861).

## Runtime errors

### Failed to create call agent, already exists for this ACS id

```

> Uncaught (in promise) CallError: CallClient.createCallAgent: Failed to create call agent, call agent for this ACS Id already exists

Reviewer: this shouldn't happen anymore right? It should be the new console warn message?

Author: They might see this error from other functions that are invoked as well like the direct create functions. I think that there is value in providing some information for whats going on here.

    at toCallError (CallContext.ts:739:16)
    at CallContext.eval (CallContext.ts:704:35)
    at Generator.throw (<anonymous>)
    at rejected (CallContext.ts:21:65)
```

> We can see this error at times when some calls to construct our `AzureCommunicationCallAdapter` or `StatefulCallClient`. This can be caused by passing the arguments to these functions more than once or too often.

Reviewer 2: ```suggestion
This error can occur when constructing `AzureCommunicationCallAdapter` or `StatefulCallClient`, often caused by passing arguments more than once or too frequently to these functions.
```
